docs/troubleshooting.md

```diff
@@ -1,8 +1,149 @@
+
+3. Check for errors in the backup:  
+    `oc describe backup <backupName>`
+
+    - If you have a local Velero installation, you can also run:  
+     `velero describe backup <backupName> -n <OADP-namespace>` and `velero backup logs <backupName> -n <OADP-namespace>`
+
+4. Check for errors in the VolSync pod:  
+    `oc get po -n openshift-operators`  
+    
+    `oc logs <volsync-pod> -n openshift-operators`  
+
+5. Check the details of volumeSnapshotContent, volumeSnapshot, and PVC, from volumeSnapshotMover:  
+    `oc get <resource> -n <OADP-namespace>`
+
+    `oc describe <resource> <resource-name> -n <OADP-namespace>`
+
+6. Fix errors if any. 
+
+If the issue still persists, [create a new issue](https://github.com/konveyor/volume-snapshot-mover/issues/new) if [an issue doesnt exist already](https://github.com/konveyor/volume-snapshot-mover/issues)
+
+
+<hr style="height:1px;border:none;color:#333;">
+
+<h1 align="center">Debugging Failed VolumeSnapshotMover Restores<a id="restore"></a></h1>
+
+This section includes how to debug a failed restore using volumeSnapshotMover.
+
+1. Check for errors in volumeSnapshotMover controller logs:  
+
+    `oc logs <vsm-pod> -n <OADP namespace>`  
+
+2. Check for errors in the Velero pod:  
+
+    `oc logs <velero-pod> -n <OADP namespace>`   
+
+3. Check for errors in the restore:  
+    `oc describe restore <restoreName>`
+
+    - If you have a local Velero installation, you can also run:    
+     `velero describe restore <restoreName> -n <OADP-namespace>` and `velero restore logs <restoreName> -n <OADP-namespace>`
+
+4. Check for errors in the VolSync pod:  
+    `oc get po -n openshift-operators`  
+    
+    `oc logs <volsync-pod> -n openshift-operators`  
+
+5. Check the details of volumeSnapshotContent, volumeSnapshot, and PVC, from volumeSnapshotMover:  
+    `oc get <resource> -n <OADP-namespace>`
+
+    `oc describe <resource> <resource-name> -n <OADP-namespace>`
+
+6. Fix errors if any. 
+
+If the issue still persists, [create a new issue](https://github.com/konveyor/volume-snapshot-mover/issues/new) if [an issue doesnt exist already](https://github.com/konveyor/volume-snapshot-mover/issues)
+
+
+<hr style="height:1px;border:none;color:#333;">
+
+<h1 align="center">Known Issues<a id="misconfig"></a></h1>
+
+<h3>Restore can take a long period of time to complete<a id="restoretime"></a></h3>
 
 - As of now, the CSI plugin blocks the restore process for each PVC that will
-    be restored via volumesnapshotmover restore. 
-    - This can create long restore times, so we are working on possible solutions.
-    
\ No newline at end of file
+    be restored via volumeSnapshotMover restore. 
+    - This can create long restore times, so we are working on possible solutions, and should be resolved in OADP 1.1.1
+    
+
+<h3>GCP object storage is not supported<a id="gcp"></a></h3>
+
+- Currently, volumeSnapshotMover only supports AWS and Azure object storage locations. 
```

Comment:
Would it make sense to add that we support s3 compatible object storage like minio, noobaa etc

Reply:
Will do